portfolio/src/main/java/com/google/sps/servlets/PollDataServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/poll-data")
+public class PollDataServlet extends HttpServlet {
+
+  private LinkedHashMap<Integer, Integer> bigfootSightings = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/bigfoot-sightings-by-year.csv"));
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
```

Comment:
this might break if someone were to update the csv file to include a blank line or was not in the proper format. Maybe include some validation here? (e.g. check that there are 2 elements in the array, etc)